lang/en/texts/better-food-choices-for-a-better-health-lets-mobilize.html

```diff
@@ -1,3 +1,8 @@
+<!-- no side column -->
+
+<div class="row">
+	<div class="text-center small-12 medium-4 columns">
```

Comment:
medium-4 will make the column 4/12 = one third of the screen on medium screens, is that intended? If not just put "small-12".

Reply:
fixed